src/OrbitBase/include/OrbitBase/Result.h

```diff
@@ -25,7 +25,17 @@ template <typename T, typename E>
 using Result = outcome::result<T, E, outcome::policy::terminate>;
 
 template <typename T>
-using ErrorMessageOr = Result<T, ErrorMessage>;
+class ErrorMessageOr : public Result<T, ErrorMessage> {
+ public:
+  using Result<T, ErrorMessage>::Result;
+};
+
+template <>
+class ErrorMessageOr<bool> : public Result<bool, ErrorMessage> {
+ public:
+  using Result<bool, ErrorMessage>::Result;
+  operator bool() = delete;
```

Comment:
Is that also deleting the const overload? Probably. I honestly don't know.

Reply:
It is according to errors I am getting but I explicitly deleted const variant anyways